tests/test_transform.py

```diff
@@ -14,7 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 import distill
+from tests import testing_utils
+from tests.data_config import DATA_DIR
+import pytest
+from core import feature_definition
```

Comment:
This is breaking the tests. Need to import starting from module namespace, i.e. `from distill.core import feature_definition`.